pkg/ruler/ruler.go

```diff
@@ -150,6 +154,12 @@ func (cfg *Config) Validate(limits validation.Limits) error {
 		return errors.Wrap(err, "invalid ruler query-frontend config")
 	}
 
+	for _, relabel := range cfg.AlertRelabelConfigs {
```

Comment:
Prometheus validates ExternalLabels in this way: https://github.com/prometheus/prometheus/blob/199d2fb492322af138ef85d08ba022066fea784c/config/config.go#L547-L557
Should that also be done here?